homeassistant/components/freebox/__init__.py

```diff
@@ -57,23 +75,15 @@ async def async_reboot(call: ServiceCall) -> None:
 
     hass.services.async_register(DOMAIN, SERVICE_REBOOT, async_reboot)
 
-    async def async_close_connection(event):
-        """Close Freebox connection on HA Stop."""
-        await router.close()
-
-    entry.async_on_unload(
-        hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_close_connection)
```

Comment:
We still want to close the connection on Home Assistant stop event, I think. The integration is not unloaded on Home Assistant stop.